src/scenekit.cs

```diff
@@ -3991,6 +3997,11 @@ interface SCNShadable {
 		[Wrap ("WeakShaderModifiers")]
 		SCNShaderModifiers ShaderModifiers { get; set; }
 
+		[Mac (12,0), iOS (15,0), TV (15,0), Watch (8,0)]
+		[NullAllowed] // by default this property is null
+		[Export ("minimumLanguageVersion", ArgumentSemantic.Retain)]
+		NSNumber MinimumLanguageVersion { get; set; }
```

Comment:
Not sure if this should be a different numeric type - and if so, which one? int, float, nint, nfloat, etc...

Reply:
Just a FYI that that would require using a `[BindAs (typeof (<numeric type>?)]`.

Reply:
hmm so is it fine as NSNumber for now? or am I misunderstanding?

Reply:
The header says:

> @abstract The minimum language version required to interpret the shadable source code (wrapped MTLLanguageVersion). Defaults to nil.

so this should be a `[BindAs (typeof (MTLLanguageVersion?)]`

Reply:
Unfortunately this is inside a protocol so BindAs is not an option, you will need to keep `NSNumber`

```
	[Protocol, Model]
	[BaseType (typeof (NSObject))]
	interface SCNShadable {
```